extensions/wikia/Wall/Wall.setup.php

```diff
@@ -275,7 +275,12 @@
 $wgRemoveGroups['bureaucrat'][] = 'threadmoderator';
 $wgAddGroups['sysop'][] = 'threadmoderator';
 $wgRemoveGroups['sysop'][] = 'threadmoderator';
-$wgAddGroups['staff'][] = 'threadmoderator';
-$wgRemoveGroups['staff'][] = 'threadmoderator';
 $wgAddGroups['helper'][] = 'threadmoderator';
 $wgRemoveGroups['helper'][] = 'threadmoderator';
+
+if ( is_array( $wgAddGroups['staff'] ) ) {
+	$wgAddGroups[ 'staff' ][] = 'threadmoderator';
+}
+if ( is_array( $wgRemoveGroups[ 'staff' ] ) ) {
+	$wgRemoveGroups[ 'staff' ][] = 'threadmoderator';
+}
```

Comment:
Same as above; this spews warnings in dev because `$wgAddGroups[ 'staff' ]` and  `$wgRemoveGroups[ 'staff' ]` are set to boolean.